benchmarks/generated_benchmark_suites.cmake

```diff
@@ -9,8 +9,6 @@
 ################################################################################
 iree_package_name(_PACKAGE_NAME)
```

Comment:
hmm... This file grows from the initial 59 LOC in https://github.com/iree-org/iree/pull/10308 to 1200 LOC in https://github.com/iree-org/iree/pull/10602/ to 1500 LOC in this PR. Given it is a generated file, I wonder if we need to keep a copy in the codebase, or if it can be generated jit when building the benchmark suites.

Reply:
We had a discussion before about generating the file during the cmake configuration. It's a little bit tricky but doable.

Right now I think it's still in the tolerated level. But  I'll spend some time revisiting some solutions.

Reply:
Could we please file an issue so we could look into this flow later?

Reply:
Filed #10928